src/core/eval.ml

```diff
@@ -47,6 +47,11 @@ let appl_to_tref : term -> term = fun t ->
 (** Abstract machine stack. *)
 type stack = term list
 
+(** FIXME: unification hints must be allowed on types, which may involve
```

Comment:
must -> should ... on types too

Reply:
This is outdated wrt yesterday's PR isn't it?